src/System.Security.Cryptography.Csp/tests/RSACryptoServiceProviderTests.cs

```diff
@@ -270,6 +277,62 @@ public static void NonExportable_Persisted()
             }
         }
 
+        [Fact]
+        public static void ImportParameters_ExponentTooBig_Throws()
+        {
+            using (var rsa = new RSACryptoServiceProvider())
+            {
+                // Verify that Unix shims and Windows Csp both throws the same exception when large Exponent imported
+                Assert.ThrowsAny<CryptographicException>(() => rsa.ImportParameters(TestData.RsaBigExponentParams));
```

Comment:
If we were throwing WindowsCryptographicException before it's fine to have ThrowsAny here.  Since it's a non-public type everyone just sees CryptographicException in catch statements. (I agree with not throwing it on !Windows, and not throwing OpenSslCryptographicException)

I was just seeing if we could make the test more stringent, since the answer's no, no worries.